### PR TITLE
Revert "Enhance Remove-Item to work with OneDrive (#14902)"

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

We have been having issues with Remove-Item -Recure -Force failing since fafc38fe5c457afbc68b379bdf9c847562e209c5.

So, reverting that commit until the issue has been addressed.

## PR Context

https://dev.azure.com/powershell/PowerShell/_build/results?buildId=76645&view=logs&j=719313b2-5d75-51d7-9f16-928d568dcba1&t=51cacb73-0bf2-5113-ddc2-bada8ccfae1e&l=114

Tests were failing with:
```
 Describing About help files can be found in AllUsers scope 
 [-] Error occurred in Describe block 0ms 
 IOException: The directory is not empty. : 'C:\Users\VssAdministrator\Documents\PowerShell\Help\' 
 IOException: Cannot remove item C:\Users\VssAdministrator\Documents\PowerShell\Help\: The directory is not empty. : 'C:\Users\VssAdministrator\Documents\PowerShell\Help\' 
 at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\HelpSystem.Tests.ps1: line 267 
 at Invoke-Blocks, D:\a\1\s\src\powershell-win-core\bin\Release\net6.0\win7-x64\publish\Modules\Pester\4.10.1\Functions\SetupTeardown.ps1: line 135 
 at Invoke-TestGroupSetupBlocks, D:\a\1\s\src\powershell-win-core\bin\Release\net6.0\win7-x64\publish\Modules\Pester\4.10.1\Functions\SetupTeardown.ps1: line 121 
 at DescribeImpl, D:\a\1\s\src\powershell-win-core\bin\Release\net6.0\win7-x64\publish\Modules\Pester\4.10.1\Functions\Describe.ps1: line 209 
```
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Comment by TravisEz13** [comment]
`PowerShell-CI-Windows-daily` is the CI where this has been failing, so let us see if this makes that pass.

**Review by rjmholt — APPROVED** [review]

**Comment by rjmholt** [comment]
@iSazonov #14902 caused an issue with `Remove-Item`, forcing us to revert it:

```
  Describing About help files can be found in AllUsers scope
    [-] Error occurred in Describe block 0ms
      IOException: The directory is not empty. : 'C:\Users\VssAdministrator\Documents\PowerShell\Help\'
      IOException: Cannot remove item C:\Users\VssAdministrator\Documents\PowerShell\Help\: The directory is not empty. : 'C:\Users\VssAdministrator\Documents\PowerShell\Help\'
      at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\HelpSystem.Tests.ps1: line 267
      at Invoke-Blocks, D:\a\1\s\src\powershell-win-core\bin\Release\net6.0\win7-x64\publish\Modules\Pester\4.10.1\Functions\SetupTeardown.ps1: line 135
      at Invoke-TestGroupSetupBlocks, D:\a\1\s\src\powershell-win-core\bin\Release\net6.0\win7-x64\publish\Modules\Pester\4.10.1\Functions\SetupTeardown.ps1: line 121
      at DescribeImpl, D:\a\1\s\src\powershell-win-core\bin\Release\net6.0\win7-x64\publish\Modules\Pester\4.10.1\Functions\Describe.ps1: line 209
```

File: src/System.Management.Automation/engine/Utils.cs [hunk 1]
```diff
@@ -1863,7 +1863,7 @@ internal static string GetFormatStyleString(FormatStyle formatStyle)
 
             if (ExperimentalFeature.IsEnabled("PSAnsiRendering"))
             {
-                PSStyle psstyle = PSStyle.Instance;
+                PSStyle psstyle = PSStyle.Instance;
```
Comment (JamesWTruher) [inline src/System.Management.Automation/engine/Utils.cs:1866]: extra white space?

Comment (iSazonov) [reply src/System.Management.Automation/engine/Utils.cs:1866]: Yes, will remove in follow PR.

**Comment by iSazonov** [comment]
I wonder how was CIs passed in the PR. Should we fix anything in CIs?

**Comment by TravisEz13** [comment]
@iSazonov No the daily CI run serially.  You should open a new PR and fix the issue in the remove-item code.

**Pull request merged**